RecoTauTag/RecoTau/interface/AntiElectronIDMVA6.h

```diff
@@ -22,7 +22,11 @@
 #include "CondFormats/EgammaObjects/interface/GBRForest.h"
 #include "DataFormats/PatCandidates/interface/Tau.h"
 #include "DataFormats/PatCandidates/interface/Electron.h"
-//#include "DataFormats/Candidate/interface/Candidate.h"
+
+#include "FastSimulation/BaseParticlePropagator/interface/BaseParticlePropagator.h"
+#include "FastSimulation/Particle/interface/RawParticle.h"
+#include "DataFormats/PatCandidates/interface/PackedCandidate.h"
+#include <TMath.h>
```

Comment:
TMath include is not needed here, please move to .cc where it's apparently needed.